docs/src/main/sphinx/connector/hive.rst

```diff
@@ -1637,10 +1637,10 @@ with Parquet files performed by the Hive connector.
       - ``true``
     * - ``parquet.optimized-writer.enabled``
       - Whether the optimized writer is used when writing Parquet files.
-        Set this property to ``true`` to use the optimized parquet writer by
+        Set this property to ``false`` to disable the optimized parquet writer by
```

Comment:
I think we can leave "by default" out

Reply:
I agree, though we have that for several other properties as well. I assume that's because the config is setting the default value for the session property, but I agree that documenting it like that is confusing. Let's follow up in a separate PR to remove that for all of them.